src/sentry/sentry_metrics/indexer/base.py

```diff
@@ -51,6 +54,19 @@ def from_string(cls: Type[KR], key: str, id: int) -> KR:
         return cls(int(org_id), string, id)
 
 
+@dataclass(frozen=True)
+class UseCaseKeyResult:
+    use_case_id: str
+    org_id: int
+    string: str
+    id: Optional[int]
```

Comment:
When is it none? Seems to always be there if this is created via `from_string()`?